DittoDiskUsage/build.gradle

```
@@ -3,10 +3,15 @@ plugins {
     alias libs.plugins.org.jetbrains.kotlin.android
 }
 
+
+ext.libraryGroupId = "live.ditto.dittodiskusage"
+ext.libraryVersion = "0.0.1"
```

Access in Sonatype is based on the group ID. It would be easier to add new artifacts if we use our existing `live.ditto` as the group and then the library-specific name as the artifact ID (e.g. `diskusage`). This would result in full maven coordinates of `live.ditto:diskusage:<version>`. 

```suggestion
ext.libraryGroupId = "live.ditto"
ext.libraryArtifactId = "diskusage"
ext.libraryVersion = "0.0.1"
```

This depends on adding a `libraryArtifactId` extra var to `deploy.gradle`

Otherwise, I think we have to register a new organization and [verify that we own](https://central.sonatype.org/publish/requirements/coordinates/) the https://dittodiskusage.ditto.live domain.